sunpy/image/transform.py

```diff
@@ -148,3 +147,27 @@ def affine_transform(image, rmatrix, order=3, scale=1.0, image_center=None,
             rotated_image += im_min
 
     return rotated_image
+
+
+def _get_transform_method(method, use_scipy):
+    # This is re-used in affine_transform and GenericMap.rotate
+    supported_methods = {'scipy', 'skimage'}
+    if method not in supported_methods:
+        raise ValueError(f'Method {method} not in supported methods: {supported_methods}')
+
+    if use_scipy is not None:
```

Comment:
https://docs.astropy.org/en/stable/api/astropy.utils.decorators.deprecated_renamed_argument.html#astropy.utils.decorators.deprecated_renamed_argument might be easier?

Reply:
Ooh fancy, I'll give it a go!

Reply:
This wouldn't quite work, because someone could hand `use_scipy` as a positional argument and the decorator wouldn't handle that situation.

Reply:
You could probably make it work from a quick read of the docstring, but it's not worth it. What you already have works.